overlord/snapshotstate/snapshotstate_test.go

```diff
@@ -609,6 +609,7 @@ case "$*" in
     sleep 1
     ;;
 esac
+export LANG=C
```

Comment:
maybe we should add that to `testutil.MockCommand()` template?

Reply:
Yeah we could. Nothing pressing though.